users/issmirnov/tap_tog.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "issmirnov.h"
+#include QMK_KEYBOARD_H
```

Comment:
If you have the other include, you don't need this one, as it's redundant.

```suggestion
```

Reply:
Done.